source/modules/MEI.analytical.xml

```diff
@@ -245,10 +245,24 @@
   <classSpec ident="att.keySig.anl" module="MEI.analytical" type="atts">
     <desc xml:lang="en">Analytical domain attributes.</desc>
     <classes>
-      <memberOf key="att.accidental"/>
-      <memberOf key="att.keyMode"/>
       <memberOf key="att.pitch"/>
     </classes>
+    <attList>
+      <attDef ident="accid" usage="opt">
+        <gloss versionDate="2023-06-14" xml:lang="en">accidental</gloss>
+        <desc xml:lang="en">Contains an accidental for the tonic key, if one is required, <abbr>e.g.</abbr>, if <att>pname</att>
+          equals <val>c</val> and <att>accid</att> equals <val>s</val>, then a tonic of C# is indicated.</desc>
+        <datatype>
+          <rng:ref name="data.ACCIDENTAL.GESTURAL.basic"/>
+        </datatype>
+      </attDef>
+      <attDef ident="mode" usage="opt">
+        <desc xml:lang="en">Indicates major, minor, or other tonality.</desc>
+        <datatype>
+          <rng:ref name="data.MODE"/>
+        </datatype>
+      </attDef>
```

Comment:
Isn't this identical to `att.keyMode`?

Reply:
I guess than `att.keyMode` could be deleted since it is not used anywhere else?

On the other hand, `att.pitch` and `att.accidental` hold only one attribute, too.

Reply:
cf. #1726